Split sdks and nm into separate packages from @yarnpkg-pnpify (#3069)
yarnpkg/berry: packages/gatsby/content/getting-started/editor-sdks.md

Smart IDEs (such as VSCode or IntelliJ) require special configuration for TypeScript to work. This page intends to be a collection of settings for each editor we've worked with - please contribute to this list!

> The editor SDKs and settings can be generated using the `yarn sdks` (or `yarn dlx @yarnpkg/sdks` if you don't need to install it locally) command. Its detailed documentation can be found on the [dedicated page](/sdks/cli/default).

Reviewer: ```suggestion
The editor SDKs and settings can be generated using `yarn dlx @yarnpkg/sdks` (or `yarn sdks` if you added `@yarnpkg/sdks` to your dependencies). Its detailed documentation can be found on the [dedicated page](/sdks/cli/default).
```
